src/main/java/chess/model/piece/Side.java

```diff
@@ -0,0 +1,14 @@
+package chess.model.piece;
+
+public enum Side {
+    BLACK,
+    WHITE;
+
+    public boolean isWhite() {
+        return this.equals(WHITE);
+    }
+
+    public boolean isUpperSide() {
+        return this.equals(BLACK);
+    }
```

Comment:
1. enum에서 동일성이 왜 허용되는지, 현재 로직에서 동등성 혹은 동일성 중 무엇을 쓰면 좋을지 한번 공부해보시면 좋을 것 같네요.
2. 하나는 white고 하나는 upperside인데 메소드의 표현을 통일해주세요.

Reply:
1. 같은 인스턴스를 사용하기 때문에 동일성과 동등성 비교가 같다고 알고 있습니다. 어떤 것이 좋다고 생각하고 쓴 적은 없는데, 찾아보니 ==를 쓰면 아래와 같은 장점들이 있군요!
    1) provides the same expected comparison (content) as equals
    2) is more null-safe than equals()
    3) provides compile-time (static) checking rather than runtime checking
2. 통일해주었습니다!

Reply:
enum이 싱글톤인 것까지 알아가시면 좋을 것 같아요 :)